### DISCUSSION TITLE
Quick congrats to developers

### DISCUSSION THREAD

**Discussion by ThomasGorisse** [google/filament#3649] [General]
Just a quick congrats to people who made the Matrix.kt and  Vector.kt files.
It's very well written and it makes me understand (a little) again how to work with Matrix and vectors.

**Comment by romainguy** [comment]
Thank you, I'm glad you like them :) They're heavily inspired by GLSL and you can find the standalone version of that library here: https://github.com/romainguy/kotlin-math

**Reply by ThomasGorisse** [reply]
It's well documented on the README.
Maybe it could be useful to make a copy/paste of it in the TransformerManager section since I couldn't find anything information about it without moving to native code.

For people who are interested, here is also a very well written article : [https://proandroiddev.com/arcore-cupcakes-4-understanding-quaternion-rotations-f90703f3966e]()
